crates/cxx-qt-build/src/lib.rs

```diff
@@ -510,6 +510,11 @@ impl CxxQtBuilder {
             builder.flag_if_supported("-std=c++17");
             // MinGW requires big-obj otherwise debug builds fail
             builder.flag_if_supported("-Wa,-mbig-obj");
+            // macOS
+            // We need
+            //   - std::filesystem::path from 10.15
+            //   - std::shared_mutex from 10.12
+            builder.apple_deployment_target(&"10.5");
```

Comment:
does the upstream change mean we still need this ? https://github.com/rust-lang/cc-rs/pull/943 Sounds like it reads the SDK's `DefaultDeploymentTarget` again ?

Reply:
nah let's close this